dependencies/graknlabs/dependencies.bzl

```diff
@@ -26,8 +26,9 @@ def graknlabs_build_tools():
     )
 
 def graknlabs_grakn_core():
+    # TODO: revert to graknlabs
```

Comment:
Comment needed still?

Reply:
thanks! removed in https://github.com/graknlabs/client-python/pull/42/commits/c41f905dd76e35e1d746818aed30d1b01e0a0c30